Warm-up (1–10): ek variable / if-else / simple maths

Q.1 EN: Two numbers are given. Print their sum. HI: Do numbers diye hain. Unka sum print karo.

Given: a = 7, b = 5   →  print the total

#Pseudocode

1>> consider two numbers are a, b.
2>> do addition of both numbers.
3>> show the output

In [1]:
a = 7
b = 5
c = a + b
print("The Sum of two numbers is: ", c)

The Sum of two numbers is:  12


steps    a    b     c
step1    7    5      -
step2    7    5      12

print   The Sum of two number is: 12

Q2
EN: Two numbers are given. Print the larger one. HI: Do numbers diye hain. Bada wala print karo.

Given: a = 8, b = 3

Pseudocode:

1>>consider two number as a & b.
2>>take largest number, use if_else conditions.
3>>use comparison operators (>, <....)
4>>show results


In [2]:
a = 8
b = 3

lar_num = b

if a > lar_num:
   lar_num = a

if b > lar_num:
   lar_num = b

print("The larget number is: ", lar_num)
         


The larget number is:  8


steps     a     b      lar_num
step1     8     3         -
step2     8     3         3
step3     8     3         8

print   The largest number is: 8

Q3
EN: A number is given. Print its square. HI: Ek number diya hai. Uska square print karo.

Given: n = 6

#Pseudocode:

1>> already given a number
2>> consider n is number
3>>use double astric(**) sign for getting square
4>> show output

In [3]:
n = 6
square = n ** 2
print("The square is: ", square)

The square is:  36


steps   n    square
step1   6      -
step2   6      36

print   The square is: 36

Q4
EN: A number is given. Print "Even" or "Odd". HI: Ek number diya hai. "Even" ya "Odd" print karo.

Given: n = 9

#Pseudocode:

1>> we have number n.
2>> print Even or odd number
3>> n should be divisibal by 2
4>> use if else conditions
5>> print the result

In [4]:
n = 9

if n % 2 == 0:
   print(f" {n} is Even number")

else:
    print(f" {n} is Odd number")   

 9 is Odd number


steps      n  
step1      9
if         0 - false
else       0 - odd

print  9 is Odd number

Q5
EN: A number is given. Print "Positive", "Negative", or "Zero". HI: Ek number diya hai. "Positive", "Negative", ya "Zero" print karo.

Given: n = -4

Pseudocode:

1>> A given number is -4
2>> find out it's positive, negative or zero
3>> print the output

In [9]:
n = -4

if n > 0:
   print("The number is positive")

elif n < 0:
   print("The number is negative")

elif n == 0:
     print("The number is zero")      

The number is negative


Steps     n  
step1     -4
if        false
if        True 

print    The number is negative


Q6
EN: A Celsius temperature is given. Print it in Fahrenheit. (F = C * 9/5 + 32) HI: Celsius diya hai. Fahrenheit mein print karo.

Given: c = 100

#Pseudocode:
1>> celsius temperature is given
2>> c = 100
3>> Fahrenheit (F = C * 9/5 + 32)
4>> show the output


In [11]:
c = 100

if c:
   F = c* 9/5 + 32
   print(F)




212.0


steps     c      F
step1    100     -
step2    100     212
print            212

Q7
EN: A word is given. Print how many letters it has. HI: Ek word diya hai. Usme kitne letters hain print karo.

Given: word = "python"

#Pseudocode:
1>> word = python
2>> use len()
3>> show the output

In [15]:
word = "python"

length = len(word)

print("The Lenght is: ", length)

The Lenght is:  6


steps      word       length
step1      python       -
step2      python        6

print    6

Q8
EN: Two names are given. Join them with a space and print. HI: Do naam diye hain. Beech mein space laga kar jodo aur print karo.

Given: first = "Ravi", last = "Kumar"   →  "Ravi Kumar"

#Pseudocode:

1>> first & last name is given
2>> join using space
3>> print final output

In [17]:
first = "Ravi"
last = "Kumar"

print(f" The person name is: {first} {last}")

 The person name is: Ravi Kumar


steps    first      last
step1    Ravi       Kumar

print The person name is: Ravi Kumar

Q9
EN: A number is given. Print its last digit. HI: Ek number diya hai. Uska aakhri digit print karo.

Given: n = 574   →  4

#Pseudocode

1>> number is given 574
2>> use indexing
3>> print last digit
4>> show utput

In [25]:
n = 574

last_digi = str(n)
print(last_digi[2])

4


steps   n     last_digi
step1   574      -
step2   574       4

print   4

Q10
EN: Total seconds are given. Print how many full minutes and leftover seconds. HI: Total seconds diye hain. Kitne poore minutes aur bache hue seconds print karo.

Given: total = 130   →  2 minutes 10 seconds